### PR TITLE
Ehub 147

### PR DESCRIPTION
**Pull request by SACHIN-BAGHEL**

### CODE REVIEW

File: application/src/main/java/com/entando/hub/catalog/service/BundleGroupService.java [hunk 1]
```diff
@@ -120,54 +124,36 @@ public void updateMappedBy(BundleGroup toUpdate, BundleGroupController.BundleGro
...
     @Transactional
     public void deleteBundleGroup(String bundleGroupId) {
+    	logger.debug("{}: deleteBundleGroup: Delete a bundle group by id: {}", CLASS_NAME, bundleGroupId);
         Long id = Long.valueOf(bundleGroupId);
-        Optional<BundleGroup> byId = bundleGroupRepository.findById(id);
+        Optional<BundleGroup> byId = bundleGroupRepository.findById(id); //No need to fetch here from db, can be passed from controller
         byId.ifPresent(bundleGroup -> {
             deleteFromCategories(bundleGroup);
-            deleteFromBundles(bundleGroup);
+//            TODO: Delete the bundle group from Bundle Group Versions
```
Comment (nshaw) [inline application/src/main/java/com/entando/hub/catalog/service/BundleGroupService.java:147]: @SACHIN-BAGHEL is there a ticket for this or is it leftover code?

**Review by nshaw — CHANGES_REQUESTED** [review]
One open question and a minor PR incoming.

File: application/src/main/java/com/entando/hub/catalog/service/BundleGroupService.java (hunk 1 above)
Comment (AkhileshPrajapatiNV) [reply application/src/main/java/com/entando/hub/catalog/service/BundleGroupService.java:147]: @nshaw We were planing to use this code in EHUB-175 (data leak). Give us some time we will use/remove very soon,we are about to finish EHUB-175. Although this method is not in use.

Comment (nshaw) [reply application/src/main/java/com/entando/hub/catalog/service/BundleGroupService.java:147]: @AkhileshPrajapatiNV , all good, thanks.

**Review by nshaw — APPROVED** [review]

**Pull request merged**